client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+import Highlight from '~/components/Highlight/Highlight';
+import coverSvg from '~/public/images/lessons/links-and-buttons/cover.svg';
+
+const contributors = [Diana];
+const chapters = [
+  { title: '<a> element', id: 'a-element' },
+  {
+    title: 'Exemple unde folosim tag-ul a',
+    id: 'exemple-unde-folosim-tagul-a',
+  },
+  {
+    title: 'Atributul target=”_blank”',
+    id: 'atributul-target-blank',
+  },
+  // { title: 'Bune practici', id: 'bune-practici' },
+  { title: 'Întrebări BONUS', id: 'intrebari-bonus' },
+  { title: '<button> element', id: 'button-element' },
+  { title: 'Atributul type', id: 'atributul-type' },
+  { title: 'Atributul disabled', id: 'atributul-disabled' },
+  { title: 'Link-uri vs butoane', id: 'linkuri-vs-butoane' },
+  { title: 'Exerciții', id: 'exercitii' },
+];
+
+export default function LinksAndButtonsLesson() {
+  return (
+    <>
+      <Head>
+        <title> Introducere | FrontEnd.ro</title>
+        <link rel="icon" href="/favicon.ico" />
+        <SEOTags
+          title="Link-uri și butoane | Lecție HTML"
+          description="Învață să adaugi link-uri și butoane în HTML și diferența dintre acestea."
+          url="https://FrontEnd.ro/html/linkuri-si-butoane"
+        />
+      </Head>
+      <Lesson
+        id="linkuriSiButoane"
+        title="Link-uri si butoane"
+        chapters={chapters}
+        withExercises
+      >
+        <LessonContributors className="absolute" contributors={contributors} />
+        <LessonCover>
+          {/* eslint-disable-next-line react/no-danger */}
+          <div dangerouslySetInnerHTML={{
+            __html: coverSvg,
+          }}
+          />
+        </LessonCover>
+        <p>
+          Deși la prima vedere pare un subiect simplu, sunt totuși multe de
+          știut despre link-uri și butoane în HTML. În acest articol vom discuta
+          despre markup, atribute și cele mai bune practici. Totodată vom trata
+          împreună cele două concepte pentru că de multe ori se folosesc
+          link-uri în loc de butoane și invers. Astfel la sfârșitul articolului
+          vom ști exact cum să implementăm un UX corect folosind cele 2
+          elemente.
+        </p>
+        <section>
+          <h2> The Basics </h2>
+          <LessonHeading as="h3" id="a-element">
+            {'<a> element'}
+          </LessonHeading>
+          <p>
+            Elementul
+            {' '}
+            <strong className="formatted">{'<a>'}</strong>
+            {' '}
+            denumit și
+            {' '}
+            <i>ancoră</i>
+            {' '}
+            sau
+            {' '}
+            <i>link</i>
+            {' '}
+            este unul din cele mai uzuale elemente
+            folosite pentru construirea paginilor web. Dacă vrem să ne mutam la
+            o altă pagină sau la o altă zonă în cadrul aceleiași pagini, atunci
+            vom folosi acest element. Modalitatea prin care adăugăm un link
```

Comment:
We don't add the link via the `href` attribute, we specify where the link points (destination)